core/bin/migrate_database

```diff
@@ -1,7 +1,51 @@
-#!/usr/bin/env python
-"""Run any new migrations from the database"""
-import startup  # noqa: autoflake
+#!/usr/bin/env sh
+# Run any new migrations from the database
 
-from core.scripts import DatabaseMigrationScript
+TARGET="head"
+DIRECTION="upgrade"
 
-DatabaseMigrationScript().run()
+# Attempt to find the alembic.ini file
+# Ansible generally runs from within the <root>/bin directory
+INI_FILE="alembic.ini"
+SRC_DIR="."
+FOUND_INI=0
+for i in 1 2 3; do
+    if [ -f "$INI_FILE" ]; then
+        FOUND_INI=1
+        break
+    else
+        # Go up one directory to try and find the ini
+        INI_FILE="../${INI_FILE}"
+        SRC_DIR="../${SRC_DIR}"
+    fi
+done
```

Comment:
I'm not keen on searching like this. Is there a way we can explicitly provide a config file path?

Reply:
I could change this to just be `../alembic.ini`

Reply:
I wonder if we would be better off keeping the `migrate_database` script in python, rather then bringing a shell script into it. Like we are doing with library registry. But that is just a minor nit on my part, I prefer debugging and troubleshooting python scripts over shell scripts any day.